### DISCUSSION TITLE
Diablo II & LOD Expansion

### DISCUSSION THREAD

**Discussion by EGWolfwood** [diasurgical/DevilutionX#5207] [Q&A] [answered]
I love what you have done to breath life back into Diablo, It's Awesome! 💯 
I just wanted to ask if there is any chance that Diablo II & Lodr of Distruction will be next?
Sorry if I missed this in another post.

Thanks :)

**Comment by AJenbo** [comment, marked as answer]
We have no intent to look at any other applications.

**Reply by EGWolfwood** [reply]
Thank you for your quick answer to my question. :)
